File: 1119. Remove Vowels from a String.py
class Solution:
    def removeVowels(self, s: str) -> str:
        temp =  [char for char in s]
        #print(temp)
        #print(temp)
        ans = ""
        for x in range(len(temp)):
            if temp[x]!='a' and temp[x]!='e' and temp[x]!='i' and temp[x]!='o' and temp[x]!='u':
                ans+=str(temp[x])
        return ans

File: go_solution_set1/1119. Remove Vowels from a String.go
func removeVowels(s string) string {
    ans := ""
    for _, x:= range s{
        if x!='a' && x!='e' && x!='o' && x!='i' && x!='u'{
            ans+= string(x)
        }
    }
    return ans
}
